modules/hivemq-edge-module-modbus/src/main/java/com/hivemq/edge/adapters/modbus/ModbusProtocolAdapterInformation.java

```diff
@@ -58,7 +58,7 @@ protected ModbusProtocolAdapterInformation() {
 
     @Override
     public List<ProtocolAdapterConstants.TAG> getTags() {
-        return List.of(ProtocolAdapterConstants.TAG.TCP, ProtocolAdapterConstants.TAG.ETHERNET);
+        return List.of(ProtocolAdapterConstants.TAG.TCP);
```

Comment:
+ Web?

Reply:
This comment is referring to the correctly of the ModBus adapter - the WEB tag has been added as a simplified classification for what are considered web (internet) protocols. This is a loose grouping, so im not attached to keeping WEB if people think otherwise.